pipeline/outputs/stackdriver.md

```diff
@@ -184,7 +184,17 @@ Do following check:
 * If the log entry does not contain the local_resource_id field, does the tag of the log match for format?
 *   If tag_prefix is configured, does the prefix of tag specified in the input plugin match the tag_prefix?
 
-    **Other implementations**
+### caught signal (SIGSEGV)
+
+> Github reference: [#7552](https://github.com/fluent/fluent-bit/issues/7552)
+
+When the number of Workers is greater than 1, fluent-bit may interimittently crash.
+
+```
+[2023/06/07 08:15:30] [engine] caught signal (SIGSEGV)
```

Comment:
Similarly to the title, this could be anything and isn't specific to this crash with the workers so I would recommend not including it here.

Reply:
Probably say "you may see this" or something similar. However it could be any crash really.